data/mods/Magiclysm/enchanted/enchanted_melee.json

```diff
@@ -754,5 +754,176 @@
+    "plural": "Technomancer toolbars",
+    "\\": "Combat stats based off crowbar +1, techniques off Halligan bar(closest non-magical substitute), weight based on 75% of hammer & wrench(since their handles aren't included) + 100% crowbar with benefit from +1 enchantment",
+    "description": "This staff incorporates a sturdy cresent wrench on top of a prybar and a hammer on the other in a convienent package. There is a Technomancer rune embedded in the hammerhead.",
+    "weight": 1536,
+    "volume": "1750 ml",
+    "price": 3900,
+    "to_hit": 1,
+    "bashing": 15,
+    "cutting": 1,
+    "material": [ "steel" ],
+    "symbol": "/",
+    "color": "light_gray",
+    "techniques": [ "WBLOCK_1", "BRUTAL", "SWEEP" ],
+    "qualities": [ [ "PRY", 3 ], [ "HAMMER", 3 ], [ "HAMMER_FINE", 1 ], [ "WRENCH", 2 ], [ "WRENCH_FINE", 2 ] ],
+    "flags": [ "DURABLE_MELEE", "BELT_CLIP", "TRADER_AVOID", "MAGIC_FOCUS" ]
+  },
+  {
+    "id": "rune_magus_weapon",
+    "type": "GENERIC",
+    "name": "Magus staff",
+    "plural": "Magus staves",
+    "\\": "The classical vision of the wizard gets a classic weapon. DPS Stats are between the regular and ironshod versions. Stats are slightly worse than the QS+1 due to the glass weight slowing attack speed.",
+    "description": "A quarterstaff with runes carved into it and two glass jars, heat-tempered and infused with mana for durability, to act as mana receptacles. There are two Magi runes embedded at the tips.",
+    "weight": 1530,
+    "volume": "1500 ml",
+    "price": 3900,
+    "to_hit": 2,
+    "bashing": 20,
+    "material": [ "wood", "diamond" ],
+    "symbol": "/",
+    "color": "brown",
+    "techniques": [ "WBLOCK_2", "RAPID", "SWEEP" ],
+    "flags": [ "DURABLE_MELEE", "SHEATH_SPEAR", "TRADER_AVOID", "MAGIC_FOCUS" ]
+  },
+  {
+    "id": "rune_kelvinist_weapon",
+    "type": "GENERIC",
+    "name": "Kelvinist flamberge",
+    "plural": "Kelvinist flamberges",
+    "\\": "Flame-bladed sword covers half of Kelvinist's heat-cold theme. Combat stats equivalent to cutlass +1 with BRUTAL technique.",
+    "description": "A sword with an undulating blade, reminiscent of a flame. There is a Kelvinist rune embedded in the pommel.",
+    "weight": 1530,
+    "volume": "1250 ml",
+    "price": 3900,
+    "to_hit": 3,
+    "bashing": 7,
+    "cutting": 29,
+    "material": [ "wood", "steel" ],
+    "symbol": "/",
+    "color": "gray",
+    "techniques": [ "WBLOCK_2", "BRUTAL" ],
+    "qualities": [ [ "CUT", 1 ], [ "BUTCHER", 9 ] ],
+    "use_action": "WEATHER_TOOL",
+    "flags": [ "DURABLE_MELEE", "SHEATH_SWORD", "TRADER_AVOID", "MAGIC_FOCUS", "THERMOMETER" ]
+  },
+  {
+    "id": "rune_stormshaper_weapon",
+    "type": "GENERIC",
+    "name": "Stormshaper axe",
+    "plural": "Stormshaper axes",
+    "\\": "Equivalent to copper ax +1, with volume reduction so it can be sheathed like the other school weapons. Metals chosen for being two most conductive metals + balancing reasons",
```

Comment:
```suggestion
    "//": "Equivalent to copper ax +1, with volume reduction so it can be sheathed like the other school weapons. Metals chosen for being two most conductive metals + balancing reasons",
```